src/pages/index.html

```diff
@@ -0,0 +1,107 @@
+      </svg>
+    </h2>
+    <ul>
+      <li><a class="nav_link">Home</a></li>
+      <li><a class="nav_link">Moon</a></li>
+    </ul>
+  </nav>
+  <main class="main">
+    <section class="primary_section">
+      <div class="squares">
+        <div class="square_primary"></div>
+        <div class="smaller_squares">
+          <div class="square_1"></div>
+          <div class="square_2"></div>
+        </div>
+        <div class="buttons">
+          <button class="cover_button active_cover_button" type="button"></button>
+          <button class="cover_button" type="button"></button>
+          <button class="cover_button" type="button"></button>
+        </div>
+      </div>
+      <div class="computer_row">
+        <h2 class="primary_title title">The purpose of the application</h2>
+        <p class="primary_paragraph">
+          Lorem Ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+          minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+          aliquip ex ea commodo consequat. Duis aute irure dolor in
+          reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+          pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+          culpa qui officia deserunt mollit anim id est laborum.
+        </p>
+      </div>
+    </section>
+
+    <section class="secondary_section">
+      <div class="square_second"></div>
+      <div class="columns">
+        <h2 class="secondary_title title">Our favourite planet</h2>
+        <p class="secondary_paragraph">
+          Lorem Ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+          ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+          aliquip ex ea commodo consequat. Duis aute irure dolor in
+          reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+          pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+          culpa qui officia deserunt mollit anim id est laborum.
+        </p>
+      </div>
+    </section>
+  </main>
+
+  <section class="newsletter">
+    <h2 class="newsletter_title">Get notifed of my future works</h2>
+    <p class="newsletter_paragraph">
+      Subscribe to get notifed when the new repository is created on my GitHub
+      page.
+    </p>
+    <div class="form">
+      <input type="email" class="form_email" placeholder="your@email.com" />
+      <button class="form_button">
+        Subscribe
+        <img src="../media/subscribe_icon.svg" alt="subscribe_icon" />
+      </button>
+    </div>
+  </section>
+
+  <footer class="footer">
+    <ul class="footer_list">
+      <a href="https://github.com/Matthewline"><i class="fab fa-github"></i></a>
+      <a href="https://www.linkedin.com/in/mateusz-chmielewski-3b323b217/"><i class="fab fa-linkedin"></i></a>
```

Comment:
Here you introduce 3rd way of adding icons to your project. Using the `i` tag for icons is not a valid use, even though it's quite popular nowadays.